docs/Fields.md

```diff
@@ -1106,6 +1106,132 @@ export const PostShow = (props) => (
 );
 ```
 
+## Translatable Fields
+
+You may have fields which are translated in multiple languages and want users to verify each translation. To display them, you can use the `<TranslatableFields>` components which expect the translatable fields to have the following structure:
+
+```js
+{
+    name: {
+        en: 'The english value',
+        fr: 'The french value',
+        tlh: 'The klingon value',
+    }
+}
```

Comment:
I'd add a `description` field as well to make the following snippet self explanatory

Reply:
Done